### PR TITLE
fix: scraping user albums (chevereto)

### PR DESCRIPTION
**Pull request by Barbarella6666666**
I found an edge case where CDL wouldn't download the albums if they were created by the admin. I dind't include a test for this case, because it would take forever to run. The url in question is: https://imglike.com/admin/albums/?sort=likes_desc&page=1

### CODE REVIEW

File: cyberdrop_dl/crawlers/_chevereto.py [hunk 1]
```diff
@@ -66,7 +66,8 @@ async def fetch(self, scrape_item: ScrapeItem) -> None:
         if self.is_subdomain(scrape_item.url):
             return await self.direct_file(scrape_item)
 
-        match scrape_item.url.parts[1:]:
+        parts = scrape_item.url.parts[1:] if scrape_item.url.parts[1:][0] != "admin" else scrape_item.url.parts[2:]
+        match parts:
```
Comment (NTFSvolume) [inline cyberdrop_dl/crawlers/_chevereto.py]:
```suggestion
        match scrape_item.url.parts[1:]:
```

**Review by NTFSvolume — CHANGES_REQUESTED** [review]
Admin is just an user. You should generalize so it works with any user.

ex: https://imglike.com/bonton18/albums

File: cyberdrop_dl/crawlers/_chevereto.py
Comment (NTFSvolume) [inline cyberdrop_dl/crawlers/_chevereto.py:80]:
```suggestion
case [_, "albums"]:
    return await self.profile(scrape_item)
case [_]:
    return await self.profile(scrape_item)
```
